README.md

```diff
@@ -56,4 +67,24 @@ val cachingPath = cachingFilesystem.convertToCachingPath(path)
 // Continue your code using cachingPath...
 ```
 
+## Releasing:
+1. Make and checkout a release branch on github.
+2. Change the version in gradle.properties to a non-SNAPSHOT version.
+3. Update the CHANGELOG.md for the impending release.
+4. Run `git commit -am "Release AttributeCachingFileSystem X.Y.Z."` (where X.Y.Z is the new version) in the terminal or
```

Comment:
```suggestion
4. Run `git commit -am "Release X.Y.Z."` (where X.Y.Z is the new version) in the terminal or
```

Reply:
I was just following the conventions I did for my detekt extensions and for our own internal projects though

Reply:
We do this for core-libraries because we release multiple projects from it. It doesn't make a lot of sense for something like this. I didn't think about it at the time for [DetektExtensions](https://github.com/pkware/detektExtensions/pull/2#discussion_r897218763) because I was caught up on the other use of copy/paste I disagreed wiith.

Reply:
okay i will do this